src/cloud/lib/i18n/zhCN.ts

```diff
@@ -476,6 +478,16 @@ const zhTranslation: TranslationSource = {
 
   [lngKeys.GeneralYes]: '是的',
   [lngKeys.GeneralNo]: '不',
+
+  [lngKeys.ModalUnlockDashboardsTitle]: 'Unlock Dashboards',
+  [lngKeys.ModalUnlockDashboardsDescription]:
+    'Create unlimited dashboards to organize your docs, tasks.. in order to guide your workflow in your own way! Display the information the way you want to see it.',
```

Comment:
`your docs, tasks.. in order to ...` -> `your docs, tasks, etc., in order to ...`